debian/kolibri.scripts-common

```diff
@@ -284,6 +284,9 @@ kolibri_configure()
     # Remove because we might re-register as systemd during an upgrade
     update-rc.d -f kolibri remove
 
+    KERNEL_RASPBERRY=`uname -a|grep arm`
+    [ ! -z "$KERNEL_RASPBERRY" ] && echo "This operation take some minutes in this platform"
```

Comment:
Would it make sense to give a bit of space around the message to be sure it's easier to spot?

```suggestion
    [ ! -z "$KERNEL_RASPBERRY" ] && echo "\n\n  NB! This operation take some minutes in this platform..."
```

Reply:
Done, thanks!